draft-ietf-aipref-vocab.md

```diff
@@ -46,19 +46,28 @@ informative:
 
 --- abstract
 
-This document proposes a standardized vocabulary of use cases that can be targeted when expressing machine-readable opt-outs related to Text and Data Mining (TDM) and AI training. The vocabulary is agnostic to specific opt-out mechanisms and enables declaring parties to communicate restrictions or permissions regarding the use of their digital assets in a structured and interoperable manner. It defines three key use cases—TDM, AI Training, and Generative AI Training—which can be referenced by opt-out systems to ensure consistent interpretation across different implementations.
+This document proposes a standardized vocabulary for expressing preferences related to how content is consumed by automated processing systems.
```

Comment:
Should we bring up the whole gathered vs. processed dichotomy here?

Reply:
I appreciate the edit.  It's better than mine, which was aiming for the minimum change.

I still need to consult with Paul about how we want to proceed with this.  It might be easier if we did a complete rewrite (as you have done) than pick at the edges.  However, that reduces the traceability of our changes.